_topics/en/quantum-resistance.md

```diff
@@ -0,0 +1,117 @@
+    url: /en/newsletters/2020/04/29/#what-are-the-potential-attacks-against-ecdsa-that-would-be-possible-if-we-used-raw-public-keys-as-addresses
+
+  - title: "Question: could taproot create security risk from quantum threats?"
+    url: /en/newsletters/2020/02/26/#could-taproot-create-larger-security-risks-or-hinder-future-protocol-adjustments-re-quantum-threats
+
+  - title: "Question: does hashing pubkeys really provide quantum resistance?"
+    url: /en/newsletters/2019/10/30/#why-does-hashing-public-keys-not-actually-provide-any-quantum-resistance
+
+  - title: "Bitcoin Core contributor meeting transcripts: taproot quantum discussion"
+    url: /en/newsletters/2019/06/12/#taproot-accumulator-quantum
+
+  - title: BIP151 discussion, including about NewHope quantum-resistant key exchange
+    url: /en/newsletters/2018/09/11/#bip151-discussion
+
+  - title: PR opened for initial BIP151 support, NewHope quantum resistance to follow
+    url: /en/newsletters/2018/08/28/#pr-opened-for-initial-bip151-support
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Taproot
+    link: topic taproot
+
+  - title: Version 2 P2P transport
+    link: topic v2 p2p transport
+
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Quantum resistance** is the ability for cryptographic protocols to
+  remain secure in the presence of fast quantum computers.
+
+---
+
+Bitcoin uses a variety of cryptographic protocols with varying degrees
+of vulnerability to fast quantum computers:
+
+- **SHA256, SHA256d, and RIPEMD160** used variously in Bitcoin's proof
+  of work and to uniquely identify blocks, scripts, individual
+  transactions, and collections of transactions, plus used for hash
+  locks in [HTLCs][topic htlc], would have their security strength
+  reduced to its square root by [Grover's algorithm][] running on an
+  idealized quantum computer.  That means an algorithm like SHA256 that
+  currently has an estimated second preimage resistance of 256 bits
+  (2<sup>256</sup>) would be reduced to 128 bits (2<sup>128</sup> is the
+  square root of 2<sup>256</sup>).  That loss can be overcome by using a
+  roughly equivalent algorithm with twice as many security bits, e.g.
+  going from SHA256 to SHA512.
+
+- **ECDSA** public keys used in Bitcoin are vulnerable to a
+  factorization attack using [Shor's algorithm][].  This would
+  completely eliminate the security of ECDSA, assuming an idealized
+  quantum computer.  Since public keys used for proposed [schnorr
+  signatures][topic schnorr signatures] are essentially identical to
+  those used for ECDSA, the same attack applies.  Quantum-resistant
+  alternatives to ECDSA are known, but they involve much larger key and
+  signature sizes, so most developers seem to prefer to delay upgrading
+  until it's necessary.
+
+- **Noise** is the protocol used for encrypted communication in LN.
```

Comment:
I personal hadnt heard of Noise before and went out to google it. Perhaps others would benefit from a link?

https://noiseprotocol.org/ is the protocol website, but maybe an explanation like https://duo.com/labs/tech-notes/noise-protocol-framework-intro or another would be appropriate